muikku-core-plugins/src/main/resources/fi/otavanopisto/muikku/plugins/workspace/WorkspacePluginMessages_fi.properties

```diff
@@ -39,10 +39,8 @@ plugin.workspace.discussions.pageTitle = Keskustelu
 plugin.workspace.journal.pageTitle = Oppimisp�iv�kirja
 plugin.workspace.journal.studentHasNoEntries = Opiskelijalla ei ole yht��n p�iv�kirjamerkint��
 plugin.workspace.journal.noEntries = Sinulla ei ole viel� yht��n p�iv�kirjamerkint��.
+plugin.workspace.journal.loadMoreEntries = Lataa lis�� p�iv�kirjamerkint�j�
```

Comment:
Kind of a long text for a button. Maybe just "Lataa lisää" as it should be a given that they will be journal entries?